### ISSUE TITLE
Standalone binary doesn't work

### ISSUE THREAD

**Issue by shirok** [shirok/Gauche#800]
On OSX:

```
$ ./sample-repl
*** ERROR: dynamic linking of @/gauche--collection failed: couldn't find initialization function "_Scm_Init_gauche__collection"
    While loading "@gauche/collection.sci" at line 1
    While compiling "@gauche/sequence.sci" at line 1: (define-module gauche.sequence (extend gauche.collection) (export referencer modifier subseq call-wit ...
    While loading "@gauche/sequence.sci" at line 1
    While compiling "@gauche/generator.sci" at line 1: (define-module gauche.generator (use gauche.sequence) (use gauche.partcont) (export list->generator v ...
    While loading "@gauche/generator.sci" at line 1
    While compiling "@text/console.sci" at line 1: (define-module text.console (use gauche.generator) (use gauche.sequence) (use gauche.termios) (use da ...
    While loading "@text/console.sci" at line 1
    While compiling "@gauche/interactive/editable-reader.scm" at line 1: (define-module gauche.interactive.editable-reader (use text.console) (use util.match) (use file.util) ...
    While loading "@gauche/interactive/editable-reader.scm" at line 1
    While loading "@gauche/interactive.scm" at line 1
    While compiling "(input string port)" at line 1: (use gauche.interactive)
    While loading "(input string port)" at line 1
Stack Trace:
_______________________________________
```

On Linux, it SEGVs.

**Comment by shirok** [comment]
It appears https://github.com/shirok/Gauche/commit/8c812aa748301dc5be54872a2a2cef3909b25b31 misunderstood lookup_entry set up initfn entries of prelinked modules.

**Comment by shirok** [comment]
Fixed.

**Comment by deril** [comment]
@shirok Do you know when release with this fix is planned to be made?

**Comment by shirok** [comment]
I haven't fixed the schedule for the next release.  Do you need a public release for this?

**Comment by deril** [comment]
Would be nice, if it's not hard

**Comment by shirok** [comment]
We have accrued some amount of bug fixes after 0.9.11, so I'm ok to go forward to 0.9.12.  Can't promise the hard date, but it's always good to start early.  (It always takes longer than expected).
